cms/djangoapps/contentstore/views/item.py

```diff
@@ -84,6 +84,12 @@
 
 
 highlights_setting = WaffleSwitch(u'dynamic_pacing', u'studio_course_update')
+WAFFLE_FLAG_NAMESPACE = WaffleFlagNamespace(name=u'test_settings')
+SHOW_REVIEW_RULES_FLAG = CourseWaffleFlag(
+    WAFFLE_FLAG_NAMESPACE, flag_name=u'show_review_rules',
```

Comment:
nit: if you can't fit all parameters on one line, I prefer keeping to one parameter per line. Looks more consistent, and it's harder to overlook one.

Reply:
> Why show/hide this block content in JS? Do we need to send the corresponding block's HTML to the user's browser in the situation where the waffle flag is off? Or could we "hide" it by never having sent that portion of the HTML in the first place?

Possibly. Any suggestions on the flow - i.e. where the page is being rendered?

Reply:
I think digging into how to answer such a question will turn out to be a skill that serves you well. 😄

Reply:
The latest file changes show the new code path.